client/account-recovery/lost-password/lost-password-form/index.jsx

```diff
@@ -14,83 +14,87 @@ import Card from 'components/card';
 import Button from 'components/button';
 import FormLabel from 'components/forms/form-label';
 import FormInput from 'components/forms/form-text-input';
-import { fetchResetOptionsByLogin } from 'state/account-recovery/reset/actions';
-import { getAccountRecoveryResetOptions } from 'state/selectors';
 
-export class LostPasswordFormComponent extends Component {
-	constructor() {
-		super( ...arguments );
+import {
+	fetchResetOptionsByLogin,
+	updatePasswordResetUserData,
+} from 'state/account-recovery/reset/actions';
 
-		this.state = {
-			isSubmitting: false,
-			userLogin: '',
-		};
-	}
+import {
+	isRequestingAccountRecoveryResetOptions,
+	getAccountRecoveryResetUserData,
+	getAccountRecoveryResetOptionsError,
+} from 'state/selectors';
 
+export class LostPasswordFormComponent extends Component {
 	submitForm = () => {
-		this.setState( { isSubmitting: true } );
-
-		//This is only here to test the redux action and will be replaced in a future PR
-		this.props.fetchResetOptionsByLogin( this.state.userLogin );
+		this.props.fetchResetOptionsByLogin( this.props.userLogin );
 	};
 
 	onUserLoginChanged = ( event ) => {
-		this.setState( { userLogin: event.target.value } );
+		this.props.updatePasswordResetUserData( {
+			user: event.target.value
+		} );
 	};
 
-	componentWillReceiveProps( nextProps ) {
-		if ( nextProps.resetOptions ) {
-			this.setState( { isSubmitting: false } );
-		}
-	}
-
 	render() {
-		const { translate } = this.props;
-		const { isSubmitting, userLogin } = this.state;
-		const isPrimaryButtonDisabled = ! userLogin || isSubmitting;
+		const {
+			translate,
+			userLogin,
+			isRequesting,
+			requestError,
+		} = this.props;
+
+		const isPrimaryButtonDisabled = ! userLogin || isRequesting;
 
 		return (
 			<div>
-				<h2 className="lost-password-form__title">
-					{ translate( 'Lost your password' ) }
-				</h2>
-				<p>{ translate( 'Follow these simple steps to reset your account:' ) }</p>
-				<ol className="lost-password-form__instruction-list">
-					<li>
-						{ translate(
-							'Enter your {{strong}}WordPress.com{{/strong}} username or email address',
-							{ components: { strong: <strong /> } }
-						) }
-					</li>
-					<li>
-						{ translate( 'Choose a password reset method' ) }
-					</li>
-					<li>
+				<Card compact>
+					<h2 className="lost-password-form__title">
+						{ translate( 'Lost your password?' ) }
+					</h2>
+					<p>{ translate( 'Follow these simple steps to reset your account:' ) }</p>
+					<ol className="lost-password-form__instruction-list">
+						<li>
+							{ translate(
+								'Enter your {{strong}}WordPress.com{{/strong}} username or email address',
+								{ components: { strong: <strong /> } }
+							) }
+						</li>
+						<li>
+							{ translate( 'Choose a password reset method' ) }
+						</li>
+						<li>
+							{ translate(
+								'Follow instructions and be reunited with your {{strong}}WordPress.com{{/strong}} account',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Follow instructions and be re-united with your {{strong}}WordPress.com{{/strong}} account' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).